### PR TITLE
Remove the description from the listing

### PR DESCRIPTION
**Pull request by austinEng**
-----

- [x] New helpers, if any, are documented in `helper_index.md`.
- [x] Incomplete tests, if any, are marked with TODO or `.unimplemented()`.

**[Review requirement](https://github.com/gpuweb/cts/blob/main/docs/reviews.md) checklist:**

- [x] WebGPU readability
- [x] TypeScript readability

### CODE REVIEW

File: src/common/tools/crawl.ts [hunk 1]
```diff
@@ -30,26 +29,11 @@ export async function crawl(suite: string): Promise<TestSuiteListingEntry[]> {
     const f = file.substring((suiteDir + '/').length); // Suite-relative file path
     if (f.endsWith(specFileSuffix)) {
       const filepathWithoutExtension = f.substring(0, f.length - specFileSuffix.length);
-      const filename = `../../../${suiteDir}/${filepathWithoutExtension}.spec.js`;
-
-      let mod: SpecFile;
-      if (process.env.STANDALONE_DEV_SERVER) {
-        mod = require(filename) as SpecFile;
-        // Delete the cache so that changes to the file are picked up.
-        delete require.cache[require.resolve(filename)];
```
Comment (austinEng) [inline src/common/tools/crawl.ts:39]: wait... I need to check I didn't break this..

Comment (austinEng) [reply src/common/tools/crawl.ts:39]: ok it's fine. this was just for dirtying the description

**Comment by austinEng** [comment]
#391 says that the standalone test runner was reading the descriptions. I don't think that's the case though. It isreading from `TestTreeNode` not `TestSuiteListingEntrySpec`

File: src/common/framework/file_loader.ts [hunk 2]
```diff
@@ -22,8 +23,12 @@ export abstract class TestFileLoader {
   abstract listing(suite: string): Promise<TestSuiteListing>;
   protected abstract import(path: string): Promise<SpecFile>;
 
-  importSpecFile(suite: string, path: string[]): Promise<SpecFile> {
-    return this.import(`${suite}/${path.join('/')}.spec.js`);
+  async importSpecFile(suite: string, path: string[]): Promise<SpecFile> {
+    const specFile = await this.import(`${suite}/${path.join('/')}.spec.js`);
+    assert(specFile.description !== undefined, 'Test spec file missing description: ' + path);
+    assert(specFile.g !== undefined, 'Test spec file missing TestGroup definition: ' + path);
+    specFile.g.validate();
```
Comment (kainino0x) [inline src/common/framework/file_loader.ts:30]: I think(?) moving this here prevents checking these things during presubmit. IDK if we should do it conditionally inside crawl() or if we should add another presubmit step for it.

Comment (austinEng) [reply src/common/framework/file_loader.ts:30]: I think we can do it conditionally and pass some argument to gen_listings to tell it to check or not.

Comment (kainino0x) [reply src/common/framework/file_loader.ts:30]: SGTM

**Comment by austinEng** [comment]
Already merged as part of #492

**Pull request closed without merging**